### PR TITLE
Update env get to sort environment variables before print

### PR DESCRIPTION
**Pull request by ashkan18**
# Problem
When doing 
`hokusai staging env get` the result list is not sorted which makes it hard to follow.

# Solution
Sort it 📉

### CODE REVIEW

File: hokusai/commands/env.py [hunk 1]
```diff
@@ -16,13 +16,10 @@ def delete_env(context, namespace=None):
...
   configmap.load()
-  if environment:
-    for k, v in configmap.all():
-      if k in environment:
-        print_smart("%s=%s" % (k, v))
-  else:
-    for k, v in configmap.all():
-      print_smart("%s=%s" % (k, v))
+  all_configs = configmap.all()
+  env_vars = {k:v for k,v in all_configs if (environment and k in environment) or not environment }
```
Comment (ashkan18) [inline hokusai/commands/env.py]: @izakp @saolsen is there a better way to do this? `all_configs` is a _generator_ so i couldn't call `sorted` on it, had to first iterate and create the variables and separately sort it.
This is not a big issue unless we have like a million environment variables, but if we have a million env variables we have other problems 😄

**Comment by saolsen** [comment]
🚮

File: hokusai/commands/env.py (hunk 1 above)
Comment (izakp) [reply hokusai/commands/env.py]: Haha nice one - Overoptimiztion on my part?  We can easily change the `ConfigMap.all()` method to _not_ be a generator (and if we have a million env vars yes we have other problems) by refactoring its [current implementation](https://github.com/artsy/hokusai/blob/master/hokusai/services/configmap.py#L59) to 
```
return self.struct['data'].items()
```

So we just return a dict and not an iterator 😅 https://stackoverflow.com/questions/10458437/what-is-the-difference-between-dict-items-and-dict-iteritems

Comment (izakp) [reply hokusai/commands/env.py]: Then you can easily do `sorted(configmap.all().keys())`

Comment (ashkan18) [reply hokusai/commands/env.py]: good point! did it! 🙌

**Comment by ashkan18** [comment]
@izakp this should be good to go :)

**Comment by izakp** [comment]
@ashkan18 cool - OOO and AFK this weekend but will merge Monday and get it into the next release!

File: hokusai/commands/env.py [hunk 2]
```diff
@@ -16,14 +16,10 @@ def delete_env(context, namespace=None):
...
-  if environment:
-    for k, v in configmap.all():
-      if k in environment:
-        print_smart("%s=%s" % (k, v))
-  else:
-    for k, v in configmap.all():
-      print_smart("%s=%s" % (k, v))
-
+  all_configs = configmap.all()
+  for k in sorted(all_configs.keys()):
```
Comment (izakp) [inline hokusai/commands/env.py]: More concise (`sorted` will by default to sort by the first element in a tuple):
```
for k, v in sorted(configmap.all().items()):
  if not environment or (environment and k in environment):
    print_smart("%s=%s" % (k, v))
```

**Comment by izakp** [comment]
🎉
